experiment/run_experiment.py

```diff
@@ -63,6 +64,9 @@
     'gs://{project}-backup.clusterfuzz-external.appspot.com/corpus/'
     'libFuzzer/{fuzz_target}/public.zip')
 
+# max size allowed per seed corpus for AFL
+CORPUS_ELEMENT_BYTES_LIMIT = 1 * 1024 * 1024
```

Comment:
Is this only for AFL? What happens if AFL gets a seed that is too big? Will it just ignore it?

Reply:
Yes, large inputs are ignored. Actually, this is already handled in the fuzzing worker but that is probably too late (right before invoke the fuzzer). If the runner ignores all large files, we implicitly start fuzzing experiment with no seed. 
https://github.com/Practical-Formal-Methods/fuzzbench-public/blob/1ea34bed03b288c4319f5bcc2fcda116313c8e21/experiment/runner.py#L92


But If we also check input size this in this file, we can throw error if inputs provided by user are ignored (Line 184) before we start spawning fuzzing runners.

Reply:
I see. Could we import the `CORPUS_ELEMENT_BYTES_LIMIT` constant instead of redefining it?

Reply:
fixed